apps/molgenis-components/package.json

```diff
@@ -11,6 +11,10 @@
     ".": {
       "import": "./dist/molgenis-components.es.js",
       "require": "./dist/molgenis-components.umd.js"
+    },
+    "./dist/style.css": {
+      "import": "./dist/style.css",
+      "require": "./dist/style.css"
```

Comment:
maybe not needed see: https://github.com/molgenis/molgenis-emx2/pull/1647/files

Reply:
did you try? I am quite sure it is required.

Reply:
I see you do something naughty in that PR:

```import "../../molgenis-components/dist/style.css";```

I.e. hard linking to the other module. That is then bypassing the packages. I don't like that.
You may notice in this module I do

```import "molgenis-components/dist/style.css"```

That would also work if we publish molgenis-components ever to npm (which I suppose we should do).